badge_server/main.py

```diff
@@ -448,14 +448,20 @@ def one_badge_image():
     details_link = '{}{}'.format(
         flask.request.url_root[:-1],
         flask.url_for('one_badge_target', package=package_name))
-    badge = pybadges.badge(
+
+    badge_args = dict(
         left_text=badge_name,
         right_text=_badge_status_to_text(status),
         right_color=color,
         whole_link=details_link)
 
-    response = flask.make_response(badge)
-    response.content_type = badge_utils.SVG_CONTENT_TYPE
+    if flask.request.accept_mimetypes.best_match(
```

Comment:
I haven't used flask very much - can you explain (1) what this check does and (2) why this if/else statement is better than what was there before?

Reply:
I changed it - let me know if this is more clear.

Reply:
The change makes it way more clear! Thanks!